apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -108,29 +111,41 @@ $(document).ready(function () {
 
   const showAITutorTab = canViewStudentAIChatMessages;
 
+  const showV2TeacherDashboard =
+    DCDO.get('teacher-local-nav-v2') ||
```

Comment:
should this be `DCDO.get('teacher-local-nav-v2', false)` just in case it hasn't been set yet?